onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -798,6 +798,26 @@ void OpTester::Run(
               }
             }
 
+            // Review(codemzs): Recurses only one level.
+            if (!kci) {
```

Comment:
seems duplicated with line 791 ?

Reply:
I added this while trying to get grad test working only to realize the backward expansion setup is not in place hence there is no point adding this change., I will revert.,

---
In reply to: [411621655](https://github.com/microsoft/onnxruntime/pull/3567#discussion_r411621655) [](ancestors = 411621655)